ui/apps/dev-server-ui/src/app/(dashboard)/runs/page.tsx

```diff
@@ -72,6 +72,7 @@ export default function Page() {
   const { data, fetchNextPage, isFetching } = useInfiniteQuery({
     queryKey: ['runs'],
     queryFn,
+    refetchInterval: 2500,
```

Comment:
I think this should be similar to the stream: only poll if you are not scrolled down the page. Otherwise unless you have an absolute time filter set, you will always lose the place you are at